script/check-patch-diff.js

```diff
@@ -0,0 +1,40 @@
+const { execSync, spawnSync } = require('child_process');
+const path = require('path');
+const { inspect } = require('util');
+
+const srcPath = path.resolve(__dirname, '..', '..', '..');
+const patchExportPath = path.resolve(__dirname, 'export_all_patches.py');
+const configPath = path.resolve(__dirname, '..', 'patches', 'config.json');
+
+// Re-export all the patches to check if there were changes.
+execSync(`python ${patchExportPath} ${configPath}`, { cwd: srcPath });
+
+// Update indices.
+execSync('git update-index --refresh || true');
```

Comment:
here too, use `spawnSync` and then you can ignore the exit code by just ... ignoring it :)